main.css

```diff
@@ -0,0 +1,50 @@
+@import url('https://fonts.googleapis.com/css?family=Merriweather:900|Roboto');
+
+* {
+    width: 625px;
+    color: white;
+}
+
+span {
+    border: 1px rgba(211, 211, 211, 0.2);
+    background-color: rgba(211, 211, 211, 0.2);
+    color: white;
+    border-radius: 15px;
+    margin: 10px 6px;
```

Comment:
In the line `margin: 10px 6px`, the 10px refer to both the top and bottom margin so you end up with 20px spacing between the lines of the buttons. Reduce it to 5px so that the spacing will be 10 px, as is defined in the reference.